components/table/table-wrapper.js

```diff
@@ -184,6 +185,7 @@ export class TableWrapper extends RtlMixin(LitElement) {
 			},
 			/**
 			 * Whether header row is sticky
```

Comment:
Did we want to update property descriptions like this? Not super critical but some of these arent particularly descriptive of what the property actual controls